spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/observation/AppDeployerDocumentedObservation.java

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright 2013-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.deployer.spi.app.observation;
+
+import io.micrometer.common.docs.KeyName;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.Observation.Event;
+import io.micrometer.observation.docs.DocumentedObservation;
+
+enum AppDeployerDocumentedObservation implements DocumentedObservation {
+
+	/**
+	 * Observation created upon deploying of an application.
+	 */
+	DEPLOYER_DEPLOY_OBSERVATION {
```

Comment:
The prefix of `DEPLOYER` is a bit redundant as the `AppDeployerDocumentedObservation.DEPLOY_OBSERVATION` implies that.